pgxn/neon/communicator.c

```diff
@@ -294,6 +303,15 @@ static PrefetchState *MyPState;
 
 static process_interrupts_callback_t prev_interrupt_cb;
 
+/*
+ * Array in shared memory each cell of which contains minimal in-flight request LSN sent to PS by the backend which procno is
+ * used as index in this array. This array is initially filled with InfiniteXlogRecPtr (UINT64_MAX) so if backend
+ * didn't send any request to PS, then this value doesn't effect global min.
+ *
+ * We support only 64-bit platforms and so assume that access to array elements is atomic and no any synchronization is needed. 
+ */
+static XLogRecPtr* minPrefetchLsn;
```

Comment:
Where do we clean up if the backend panics/errors with Postgres's exception-like mechanism?

Reply:
If backend panics the whole Postgres instance will be restarted. In case of other errors it will be handled by reset prefetch state mechanism (`consume_prefetch_responses` or `prefetch_on_ps_disconnect`)